tests/ui/test_ui.py

```diff
@@ -292,7 +290,7 @@ def test_keypress_autohide_users(self, view, mocker, autohide, key, widget_size)
 
         view.keypress(size, key)
 
-        view.users_view.keypress.assert_called_once_with(size, key)
+        view.right_panel.keypress.assert_called_once_with(size, key)
```

Comment:
Replacing users_view with right_panel seems good naming here, though the commit text doesn't indicate the same name?

Reply:
Oh, I change that right away.